src/components/Card/QuestionCard/Answer/Answer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useGlobal } from "context/global-context";
+
+export default function Answer({ option, answer }) {
+  const { globalDisptacher } = useGlobal();
+  //   const correctAnswerClass = answer === option ? `correct-answer` : ``;
+  //   const inCorrectAnswerClass = answer !== option ? `incorrect-answer` : ``;
```

Comment:
suggestion - avoid pushing commented code

Reply:
sure, will remove it.